internals/secrethub/audit.go

```diff
@@ -63,13 +88,28 @@ func (cmd *AuditCommand) run() error {
 		return err
 	}
 
-	tabWriter := tabwriter.NewWriter(cmd.io.Stdout(), 0, 4, 4, ' ', 0)
-	header := strings.Join(auditTable.header(), "\t") + "\n"
-	fmt.Fprint(tabWriter, header)
+	paginatedWriter, err := cmd.newPaginatedWriter(os.Stdout)
+	if err == pager.ErrPagerNotFound {
+		paginatedWriter = pager.NewFallbackPager(os.Stdout)
+	} else if err != nil {
+		return err
+	}
```

Comment:
Is there a reason for doing the fallback here or could we move this logic to the `pager.New` functionality?

Reply:
Yes, because it seemed a bit too specific. I think that we might want to handle not having a terminal pager differently in other situations. Or do you think we should handle it in `New` for now and if we ever need it we can do a small refactor?

Reply:
Okay, that's a very reasonable thought. Is there maybe a middle-ground between this being configurable and not having to do this error check outside the package? Because now we have to duplicate this check (which is very tightly coupled to the fallback mechanism) if we ever want to re-use the fallback functionality.

Some possibilities I could imagine:
```golang
paginatedWriter, err := pager.NewWithFallback(os.Stdout)
```
Or (which I like less):
```golang
paginatedWriter, err := pager.WithFallback(pager.New(os.Stdout)
```

Or maybe something similar.

Reply:
Good point! I also prefer the former solution.